Source/ForecastDeskbarView.h

```diff
@@ -18,7 +18,7 @@
 class ForecastDeskbarView : public BView
 {
 	
-public:
+public:			
```

Comment:
Remove stray tab (in the line above too)

Reply:
I updated the PR based on your comments. The main branch was updated recently, so I ditched my first  PR and created a new one.

- Icon size is now correct
- Changed menu item to sentence case
- Regenerated en.catkeys
- Removed stray tab